### PR TITLE
Add/high quality background image

### LINKED ISSUE #67451: Task: Export higher quality background images and use them for Retinas
### Details

Make sure to only load when min-width is 2048px or something like that to avoid slowing down the app.

For intro and loading steps.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

Add higher quality for background images for LiB and NL. The images are heavier but only loaded on larger screens.

## Testing Instructions

- Pull this branch and run yarn start
- Navigate to `/setup/intro?flow=link-in-bio` or `/setup/intro?flow=newsletter`
- Check if on mobile the background image for intro and processing steps has `-desktop` at the end of the file name for desktop and don't have it for mobile.
- Visually they should also look better, but could be hard to check the difference.

Related to #67451
Fixes #67451

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss [hunk 1]
```diff
@@ -30,9 +34,12 @@
 		background-attachment: fixed;
 		background-size: auto 15%, auto 15%;
 
-		@include break-large {
+		@include break-xlarge {
```
Comment (alshakero) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss]: Let's go break-huge instead. `x-large` is only 1080 and these images are huge.

**Review by alshakero — CHANGES_REQUESTED** [review]
Clean 🌵 

Just one suggestion.

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss (hunk 1 above)
Comment (renancarvalho) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss]: Cool. Next commit will address it! Thank you for checking.

**Review by alshakero — APPROVED** [review]
This does the job and works well. But I'm not very excited about this. These images are not compression-friendly at all.

**Pull request merged**